index.html

```diff
@@ -0,0 +1,33 @@
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>simpleweb</title>
+    </head>
+    <body>
+        <header>
+        <div class="toolbar">
+            <div class="logo">My Logo</div>
+            <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
```

Comment:
As you know this menu icon is part of the navigation, and it's highly recommended to put this in the `nav` semantic tag instead of `div`.